packages/medusa/src/api/routes/admin/draft-orders/update-draft-order.js

```diff
@@ -88,6 +92,8 @@ export default async (req, res) => {
       )
     }
 
+    await draftOrderService.update(draftOrder.id, value)
+
     await cartService.update(draftOrder.cart_id, value)
```

Comment:
It seems a bit risky to be passing the entire `value` arg in the doService update. Can we do like this instead: 

```
if ("no_notification_order" in value) {
  await draftOrderService.update(draftOrder.id, { no_notification_order: value.no_notification_order })
  delete value.no_notification_order
}
```